### PR TITLE
Feature Update: Implementation and Styling based on Zeplin Designs

### PR DESCRIPTION
**Pull request by Rafiduzzaman**
**Pull Request Description**:
This pull request encompasses the following accomplishments:
- Adherence to general requirements:
  - Ensured no linter errors.
  - Followed Gitflow for version control.
  - Documented work in a professional manner.

- HTML/CSS Accomplishments:
  - Followed the best practices for HTML & CSS.
  - Integrated missing JSX elements based on project requirements without adding extra functionality.
  - Styled the website to match Zeplin's design using the provided CSS rules.

---

**Code Changes**:
- Integrated missing JSX elements based on project requirements and Zeplin designs.
- Styled the website to match Zeplin's design using appropriate CSS rules.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -69,38 +56,559 @@ a {
...
+
+/* mobile version */
+@media (width >= 765px) {
+  .navItems {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    gap: 5%;
+  }
```
Comment (iambenkis) [inline src/index.css:128]: - [ ] Good job Styling your bookstore 👏. As for the page , you should make sure your design matches the design 👍. Your design does not fill all the size of the screen as in the Zeplin design , Kindly fix that 😄 

### Your design

Capture

### Zeplin

Capture0

File: src/index.css [hunk 2]
```diff
@@ -69,38 +56,559 @@ a {
...
+    background-color: var(--white-two);
+    align-items: center;
+  }
+
+  .bookDetails {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+  }
```
Comment (iambenkis) [inline src/index.css:206]: - [x] Kindly ensure that your design matches the Zeplin design to meet the requirements  👍 

### Your design

Capture

### Zeplin design

Capture0

File: src/index.css [hunk 3]
```diff
@@ -69,38 +56,559 @@ a {
...
+    letter-spacing: 1.9px;
+  }
+
+  .pageWrapper {
+    margin: auto;
+    width: 80vw;
+    background-color: #fafafa;
+    border-radius: 4px;
+    box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.2);
+  }
```
Comment (iambenkis) [inline src/index.css]: - [x] To ensure that your design matches the Zeplin design 👍 , the alignment , the margin, the padding  etc are exactly the same as in the design template

### Your design
Capture

### Zeplin design

Capture0

File: src/index.css [hunk 4]
```diff
@@ -69,38 +56,559 @@ a {
...
+    height: 1.125rem;
+    margin: 0 4rem 0 0;
+    opacity: 0.5;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 0.875rem;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: normal;
```
Comment (iambenkis) [inline src/index.css:248]: - [x] The add book section should stick to the Zeplin design , Kinldy fix that 👍 

### Your design
Capture

### Zeplin design
Capture0

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi @Rafiduzzaman 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️ Work documented professionally

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by ademibowale — APPROVED** [review]
Hi @Rafiduzzaman  , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️
- final design should match Zeplin's design (.✔️
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**